tests/integration/test_schema_avro_references.py

```diff
@@ -0,0 +1,295 @@
+"""
+karapace - schema tests
+
+Copyright (c) 2023 Aiven Ltd
+See LICENSE for details
+"""
+
+from karapace.client import Client, Result
+from tests.utils import create_subject_name_factory
+
+import json
+
+baseurl = "http://localhost:8081"
+
+# country.avsc
+SCHEMA_COUNTRY = {
+    "type": "record",
+    "name": "Country",
+    "namespace": "com.netapp",
+    "fields": [{"name": "name", "type": "string"}, {"name": "code", "type": "string"}],
+}
+
+# address.avsc
+SCHEMA_ADDRESS = {
+    "type": "record",
+    "name": "Address",
+    "namespace": "com.netapp",
+    "fields": [
+        {"name": "street", "type": "string"},
+        {"name": "city", "type": "string"},
+        {"name": "postalCode", "type": "string"},
+        {"name": "country", "type": "Country"},
+    ],
+}
+
+# job.avsc
+SCHEMA_JOB = {
+    "type": "record",
+    "name": "Job",
+    "namespace": "com.netapp",
+    "fields": [{"name": "title", "type": "string"}, {"name": "salary", "type": "double"}],
+}
+
+# person.avsc
+SCHEMA_PERSON = {
+    "type": "record",
+    "name": "Person",
+    "namespace": "com.netapp",
+    "fields": [
+        {"name": "name", "type": "string"},
+        {"name": "age", "type": "int"},
+        {"name": "address", "type": "Address"},
+        {"name": "job", "type": "Job"},
+    ],
+}
+
+SCHEMA_PERSON_INT_LONG = {
```

Comment:
`_INT_LONG` is a bit hard to know what it should be doing, I see you updated the `age` type to `long`, maybe `_AGE_LONG`

Reply:
It simple test which shows compatibility.

Reply:
fixed